cli/plasmo/templates/static/vue3/content-script-ui-mount.ts

```diff
@@ -10,14 +10,30 @@ import { createApp } from "vue"
 // @ts-ignore
 import RawMount from "__plasmo_mount_content_script__"
 
+// prettier-sort-ignore
+// @ts-ignore
+import SfcStyleContent from "style-raw:__plasmo_mount_content_script__"
+
```

Comment:
Great work with this - the usage of import scheme in these template was in my mind for a bit, nice to see it here!